rhtas/tuf-repo-init.sh

```diff
@@ -247,13 +247,15 @@ if [ "${EXPORT_KEYS:0:7}" = "file://" ]; then
   cp "${KEYDIR}/"* "${EXPORT_DIR}"
 elif [ -n "${EXPORT_KEYS}" ]; then
   echo "Exporting keys to k8s secret ${EXPORT_KEYS} ..."
+
   export AUTHDIR="/var/run/secrets/kubernetes.io/serviceaccount"
-  curl --fail -X POST \
-    --cacert "${AUTHDIR}/ca.crt" \
-    -H "Authorization: Bearer $(cat ${AUTHDIR}/token)" \
-    --header 'Content-Type: application/json' \
-    --data @- \
-    "https://kubernetes.default/api/v1/namespaces/${NAMESPACE}/secrets" <<EOF
+  export K8SCACERT="${AUTHDIR}/ca.crt"
+  export K8SSECRETS="https://kubernetes.default/api/v1/namespaces/${NAMESPACE}/secrets"
```

Comment:
`kubernetes.default` does not work in proxy-env
```suggestion
  export K8SSECRETS="https://kubernetes.default.svc/api/v1/namespaces/${NAMESPACE}/secrets"
```